.github/workflows/ci.yaml

```diff
@@ -91,16 +153,13 @@ jobs:
         run: yarn hardhat deploy
 
   export-deployments:
+    name: Export Deployments
     runs-on: ubuntu-latest
     permissions:
       contents: write
-
     steps:
       - name: Check out code
         uses: actions/checkout@v2
-        with:
-          ref: ${{ github.head_ref }}
-          token: ${{ secrets.VENUS_TOOLS_TOKEN }}
```

Comment:
This action commits a change in the last step. AFAIK we need to checkout the code with a user authorized to do that, so we would need these credentials

Reply:
1f2afb1da7e5b40bb64a103edd333259c757d72a